examples/plot_2_multidim_sigs.py

```diff
@@ -0,0 +1,249 @@
+
+freq = 10
+f_range = (5, 15)
+
+epochs = 10
+channels = 5
+
+####################################################################################################
+
+####################################################################################################
+#
+# Example 1. 2D Array (n_epochs, n_timepoints)
+# --------------------------------------------
+# The features for a 2d array with a (n_epochs, n_timepoints) organiation will be computed here.
+# This is an example of when using ``axis = None`` is appropriate, assuming each epoch was recorded
+# continuously. Data will be simulated to produce to alternating epoch types (fixation and task)
+# with varying rise-decay symmetries.
+
+# Simulate a timeseries
+sigs = np.zeros((epochs, n_seconds*fs))
+
+for idx in range(epochs):
+
+    if idx % 2 == 0:
+        # Fixation epoch
+        sim_components = {'sim_powerlaw': dict(exponent=-2),
+                          'sim_bursty_oscillation': dict(freq=10, cycle='asine',
+                                                         enter_burst=0.75, rdsym=0.5)}
+    else:
+        # Task epoch
+        sim_components = {'sim_powerlaw': dict(exponent=-2),
+                          'sim_bursty_oscillation': dict(freq=10, cycle='asine',
+                                                         enter_burst=0.75, rdsym=0.75)}
+
+    sigs[idx] = sim_combined(n_seconds, fs, components=sim_components)
+
+####################################################################################################
+
+# Compute features
+thresholds = dict(amp_fraction_threshold=0., amp_consistency_threshold=.5,
+                  period_consistency_threshold=.5, monotonicity_threshold=.8,
+                  min_n_cycles=2)
+
+compute_kwargs = {'burst_method': 'cycles', 'threshold_kwargs': thresholds}
+
+df_features = compute_features_2d(sigs, fs, f_range, axis=None,
+                                  compute_features_kwargs=compute_kwargs)
+
+####################################################################################################
+
+# Add a column specifying epoch type
+for idx, df in enumerate(df_features):
+    df['epoch'] = 'fixation' if idx % 2 == 0 else 'task'
+
+# Merge into a single dataframe
+df_concat = pd.concat(df_features, axis=0, ignore_index=True)
+
+# Limit to bursting cycles
+df_bursts = df_concat[df_concat['is_burst'] == True]
+
+# Plot
+plot_feature_categorical(df_bursts, 'time_rdsym', group_by='epoch', ylabel='Rise-Decay Symmetry',
+                         xlabel=['Fixation', 'Task'])
+
+####################################################################################################
+#
+# Here we show what would happen if axis = 0 was incorrectly used when computing the features
+# for an (n_epochs, n_timeseries) array. If bursts exist at the edges of epochs, they will be
+# incorrectly labeled as non-burst cycles. This is due to the inability to compute consistency
+# features for the first and last cycle of each signal.
```

Comment:
I found this point a bit confusing - it doesn't really describe _why_ this is the case, which is because there is a difference between computing when assuming continuity between epochs, and when not.